datafusion/src/physical_plan/aggregates.rs

```diff
@@ -122,6 +125,7 @@ pub fn return_type(
             coerced_data_types[0].clone(),
             true,
         )))),
+        AggregateFunction::ApproxQuantile => Ok(DataType::Float64),
```

Comment:
It's better to use
`AggregateFunction::ApproxQuantile => Ok(&coerced_data_types[0])` and make consistent.